cpu/samd21/Kconfig

```diff
@@ -10,6 +10,7 @@ config CPU_FAM_SAMD21
     select CPU_COMMON_SAM0
     select CPU_CORE_CORTEX_M0PLUS
     select HAS_CPU_SAMD21
+    select HAS_PERIPH_GPIO_FAST_READ
```

Comment:
this needs to be defined in `kconfigs/Kconfig.features`

Reply:
I was under the impression that I was just reusing a feature already used by a different MCU. Now I feel betrayed :disappointed: 

Added it to the feature list with a generic description.